packages/sdk/src/types/ConnectorTypes.ts

```diff
@@ -17,7 +17,7 @@ export enum MediaType {
 
 export enum ConnectorType {
     media = 'media',
-    fonts = 'fonts',
```

Comment:
Intentional change?

Reply:
I heard from @Matthiee that this was necessary and he asked me to change this in my PR we evaluated and thought it wasn't breaking

Reply:
I suggested this change because: `await SDK.connector.getAllByType('fonts');` will throw. However, I agree that this change is breaking.

Reply:
@Matthiee  Should it also be updated from Environment API type? Now it has [plural version](https://github.com/chili-publish/grafx-backend/blob/main/GraFx.Environment/CHILI.GraFx.Environment.API/CHILI.GraFx.Environment.API.Experimental/Modules/Connectors/Models/Api/ConnectorType.cs#L12) as was here before

Reply:
@brapoprod This can be reverted, it will be fixed on the engine side.